build-tools/packages/build-tools/package.json

```diff
@@ -55,6 +55,7 @@
 		"json5": "^2.2.3",
 		"lodash": "^4.17.21",
 		"lodash.isequal": "^4.5.0",
+		"picomatch": "^2.3.1",
```

Comment:
Is `glob` insufficient for this? It's already available as a dep.

Reply:
depcruise use picomatch, and it has the functionality to detect whether a string is a glob or not?

Reply:
hasMagic should do that: https://www.npmjs.com/package/glob#hasmagicpattern-string--string-options-globoptions--boolean

But if the goal is to replicate what depcruise does then just add a comment (OH WAIT IT'S JSON)